robottelo/ui/locators/base.py

```diff
@@ -1754,22 +1758,23 @@
         "//button[contains(@ng-show, 'list') and "
         "contains(@ng-click, 'removeRepositories')]"),
     "contentviews.repo_search": (
-        By.XPATH, "//input[@ng-model='repositorySearch']"),
+        By.XPATH, "//input[@ng-model='table.searchTerm']"),
```

Comment:
oh no man, I don't mind your help, but really don't need to waste your time on testing as it is really time consuming operations at that moment